Modules/CTP/src/RawDataReaderCheck.cxx

```diff
@@ -199,7 +200,12 @@ Quality RawDataReaderCheck::check(std::map<std::string, std::shared_ptr<MonitorO
         }
         if (!relativeRates) {
           fHistDifference->Scale(1. / (mCycleDuration * mFraction));
-          fHistPrev->Scale(1. / (mCycleDuration * mFraction));
+          if (fHistPrev == nullptr) {
+            std::cout << " crashing here" << std::endl;
```

Comment:
Please use `ILOG` and not `cout`. Also, I am not sure this message should really go to production. A more informative message would be better.